src/ansys/grantami/recordlists/_connection.py

```diff
@@ -23,18 +23,18 @@
 from collections import defaultdict
 import concurrent.futures
 import functools
-from typing import Iterator, List, Optional, Tuple, Union
+from typing import Iterable, Iterator, List, Optional, Tuple, Union, cast
 
-from ansys.grantami.serverapi_openapi import api, models  # type: ignore[import]
-from ansys.openapi.common import (  # type: ignore[import]
+from ansys.grantami.serverapi_openapi.v2025r2 import api, models  # type: ignore[import-not-found]
+from ansys.openapi.common import (  # type: ignore[import-not-found]
     ApiClient,
     ApiClientFactory,
     ApiException,
     SessionConfiguration,
     Unset,
     generate_user_agent,
 )
-import requests  # type: ignore[import]
+import requests  # type: ignore[import-untyped]
```

Comment:
Can we not import types-requests here and remove the suppression?

Reply:
I plan to create an in issue and handle this in a separate PR. The problem with running mypy in pre-commit by default is that it runs outside the virtual env, and so doesn't have access to any third-party dependencies. Jobqueue is set up to add mypy as a dev dependency, and then run the version of mypy in the project, which means it can use types for requests, openapi-common, and the bindings.

Reply:
https://github.com/ansys/grantami-recordlists/issues/376